src/components/NFTPage/Recommendation.vue

```diff
@@ -237,7 +240,19 @@ export default {
         });
       }
 
-      return recommendedList.slice(0, DISPLAY_ITEM_COUNT);
+      let crossSellList = [];
+      const crossSellItems = CROSS_SELL_PRODUCT_IDS_MAP[this.classId] || [];
+      if (crossSellItems.length > 0) {
+        crossSellList = crossSellItems.map(classId => ({ classId }));
+      } else {
+        return recommendedList.slice(0, DISPLAY_ITEM_COUNT);
+      }
+      const combinedList = [...crossSellList, ...recommendedList];
+      const uniqueList = Array.from(
+        new Map(combinedList.map(item => [item.classId, item])).values()
+      );
+
+      return uniqueList.slice(0, DISPLAY_ITEM_COUNT);
```

Comment:
can we still `return recommendedList.slice(0, DISPLAY_ITEM_COUNT);`  at the end but alter recommendedList when there are cross-sell items